libraries/SITL/SITL.cpp

```diff
@@ -403,6 +403,14 @@ Vector3f SITL::convert_earth_frame(const Matrix3f &dcm, const Vector3f &gyro)
     return Vector3f(phiDot, thetaDot, psiDot);
 }
 
+float SITL::get_rangefinder(uint8_t rot) {
```

Comment:
Could we use `enum Rotation` here?

Reply:
I couldn't cast the AP_int8 rangefinder orientation param to the enum, but maybe you know a way?

Reply:
Throw in a `.get()`

Reply:
That fixed it, thanks